nemo_text_processing/text_normalization/normalize.py

```diff
@@ -303,24 +303,22 @@ def _split_tokens_to_reduce_number_of_permutations(self, tokens: List[dict]) ->
         assert sum([len(s) for s in splits]) == len(tokens)
         return splits
 
-    def normalize(
-        self, text: str, verbose: bool = False, punct_pre_process: bool = False, punct_post_process: bool = False
-    ) -> str:
+    def normalize(self, text: str, punct_pre_process: bool = False, punct_post_process: bool = False) -> str:
```

Comment:
this change is not backward compatible with some of the pretrained tts checkpoints that have `verbose` arg in normalize(). Could you introduce a new arg in the init instead of reusing `verbose`? maybe `exit_on_failure` or smth?